rules.adoc

```diff
@@ -401,19 +409,19 @@ For Lightweight-specific regulations see <<regulations-inference-in-lightweight>
 
 A team claiming that their robot is affected by the other team’s robot in any
 way must show the proof/evidence of the interference. Any interference needs to
-be confirmed by an OC member if a claim is placed by the other team.
+be confirmed by the tournament organizers if a claim is placed by the other team.
 
 [[robots-control]]
 === Control
 
-The use of remote control of any kind is not allowed during the match.  Robots
+The use of remote control of any kind is not allowed during the match. Robots
 must be started and stopped manually by humans and be controlled autonomously.
 
 [[communication]]
 === Communication
 
-Robots are not allowed to use any kind of communication during game play unless
-the communication between two robots is via Bluetooth class 2 or class 3
+Robots are not allowed to use any kind of communication during gameplay unless
```

Comment:
Note that this comes verbatim from the RCJ General Rules (I guess we should be updating that too at some point: https://junior.robocup.org/robocupjunior-general-rules/)